BAZProject/HomeMovieViper/View/HomeViewCollectionsExtension.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  HomeViewCategoriesCollectionExtension.swift
+//  BAZProject
+//
+//  Created by 1050210 on 31/01/23.
+//
+
+import UIKit
+
+// MARK: - CollectionView's DataSource
+
+extension HomeMoviesView: UICollectionViewDataSource{
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if collectionView == self.categoriesMoviesCollectionView{
+            return presenter?.getCategoriesMoviesCount() ?? 0
+        }else{
+            return presenter?.getMoviesCount() ?? 0
+        }
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        if collectionView == self.categoriesMoviesCollectionView{
+            guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "CategoriesMoviesCollectionViewCell", for: indexPath) as? CategoriesMoviesCollectionViewCell
+            else { return UICollectionViewCell() }
+            
+            presenter?.getCategorieImage(index: indexPath.row, completion: { categoryImage in
+                cell.setupCategoryImage(for: categoryImage ?? UIImage())
+            })
+            cell.delegate = self
+            cell.setupCell(indexPath: indexPath.row)
+            
+            return cell
+        }else{
+            guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "MovieCollectionViewCell", for: indexPath) as? MovieCollectionViewCell else { return UICollectionViewCell() }
+            
+            presenter?.getImage(index: indexPath.row, completion: { imageMovie in
+                cell.setupMovieImage(for: imageMovie ?? UIImage())
+            })
+            return cell
+        }
+    }
+}
+
+// MARK: - CollectionView's Delegate
+
+extension HomeMoviesView: UICollectionViewDelegate{
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, insetForSectionAt section: Int) -> UIEdgeInsets {
+        return UIEdgeInsets(top: insets,
+                            left: insets,
+                            bottom: insets,
+                            right: insets)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+        return CGFloat(8.0)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+        return minimumInterItemSpacing
+    }
+}
+
+extension HomeMoviesView: UICollectionViewDelegateFlowLayout{
+    internal func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+
+        if collectionView == self.categoriesMoviesCollectionView{
```

Comment:
Remember to keep an standard on your code, the spaces before the `{` and after the `:`